daisy_workflows/build-publish/rhui/rhua_artifacts/health-check.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Executes RHUA health check
```

Comment:
nit but description field is really more of a name field:
```
This may be used by systemd (and other UIs) as a user-visible label for the unit,
so this string should identify the unit rather than describe it, despite the name.
```